onnxruntime/test/framework/inference_session_test.cc

```diff
@@ -2068,19 +2068,9 @@ TEST(InferenceSessionTests, TestParallelExecutionWithCudaProvider) {
   so.session_logid = "InferenceSessionTests.TestParallelExecutionWithCudaProvider";
   InferenceSession session_object{so, GetEnvironment()};
 
-  ASSERT_STATUS_OK(session_object.RegisterExecutionProvider(DefaultCudaExecutionProvider()));
-
-  ASSERT_STATUS_OK(session_object.Load(model_uri));
-
-  auto status = session_object.Initialize();
-
-  ASSERT_TRUE(status.IsOK());
-
-  const auto& so_queried = session_object.GetSessionOptions();
+  auto status = (session_object.RegisterExecutionProvider(DefaultCudaExecutionProvider()));
 
-  // execution mode is sequential since we have registered the CUDA EP
-  // (which isn't supported by the parallel execution mode)
-  ASSERT_TRUE(so_queried.execution_mode == ExecutionMode::ORT_SEQUENTIAL);
+  ASSERT_FALSE(status.IsOK());
```

Comment:
Is there any point to keeping this test?

Reply:
you are right, this test will be covered with checksessionoption test.